src/lib/storage.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
```

Comment:
Good storage module, again, try to leave out comments before you submit, i.e., you could have a new branch set up that is only meant for your comments since I know you need them for yourself to understand what's going on.